tests/testthat/model-examples/1/grid.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
-
```

Comment:
This is fine. This file is created when the model is run on the grid, which apparently it was last time and wasn't this time. Doesn't matter for our tests.